components/admin_console/data_retention_settings/data_retention_settings.tsx

```diff
@@ -99,9 +103,23 @@ export default class DataRetentionSettings extends React.PureComponent<Props, St
                 className: 'actionIcon',
             },
         ];
+        if (this.includeBoards) {
+            columns.splice(columns.length - 1, 0,
+                {
+                    name: (
+                        <FormattedMessage
+                            id='admin.data_retention.globalPoliciesTable.boards'
+                            defaultMessage='Boards'
+                        />
+                    ),
+                    field: 'boards',
+                },
+            );
+        }
+        return columns;
     }
     getCustomPolicyColumns = (): Column[] => {
-        return [
+        const columns = [
```

Comment:
nit: I guess you changed it for consistency reasons, but still, the style is not consistent. I would suggest to change the other method constant initialization to the format here, I mean `const columns = [` with the `[` in the same line.